__tests__/app.test.js

```diff
@@ -342,4 +342,36 @@ describe("GET /api/users", () => {
     .get('/api/userss')
     .expect(404)
   })
+})
+
+describe('GET /api/articles?sort_by=', () => {
+  test("200: Responds with an array sorted by the sort_by Query & the order we want it. In this case DESCENDING", () => {
+    return request(app)
+    .get('/api/articles?sort_by=title&order=DESC')
+    .expect(200)
+    .then(({body}) => {
+      const {articles} = body
+      expect(articles).toBeSortedBy('title', {descending: true})
+    })
+  })
+
+  test("200: Responds with an array sorted by the sort_by Query & the order we want it. In this case ASCENDING", () => {
+    return request(app)
+    .get('/api/articles?sort_by=title&order=ASC')
+    .expect(200)
+    .then(({body}) => {
+      const {articles} = body
+      expect(articles).toBeSortedBy('title')
```

Comment:
I would add descending:false to this assertion to prove it's in ascending order!